src/ErrorUtil/validateErrors.js

```diff
@@ -0,0 +1,36 @@
+const validateError = (
+  targetCase,
+  supportedCases,
+  erorrMessages,
+  originalText,
+) => {
+  const error = {
+    errors: [],
+  };
+
+  const isSupportedCase = supportedCases.includes(targetCase);
+
+  if (targetCase) {
+    if (!isSupportedCase) {
+      error.errors.push({
+        message: erorrMessages.invalidCase,
+      });
+    }
+  } else {
+    error.errors.push({
+      message: erorrMessages.noToCaseParam,
```

Comment:
Avoid nested `if` and `else` statements

Reply:
ok, but how I can do it in this case. I tried switch case but it doesn't work because I must have an array of errors i they are, not only one.

Reply:
> ok, but how I can do it in this case. I tried switch case but it doesn't work because I must have an array of errors if they are present, not only one.

```javascript
  if (!targetCase) {
    error.errors.push({
      message: erorrMessages.noToCaseParam,
    });
  }
   
  if (!isSupportedCase) {
    error.errors.push({
      message: erorrMessages.invalidCase,
    });
  }
```